pingcap/tidb-binlog
# Add unique key info in slave proto

### What problem does this PR solve? 
Add unique key info in slave proto

### What is changed and how it works?
Add unique key info in slave proto
need https://github.com/pingcap/tidb-tools/pull/310 merged first
### Check List 

Tests 

 - Unit test
 - Manual test (add detailed scripts or steps below)
```
create aa(id int);
create table a3(id varchar(255) primary key, c2 int, c3 int, unique key idx1(c3,c2));
create table a3(id int primary key, c2 int, c3 int, unique key idx1(c3,c2));

//and insert data and check the proto works as exptected
```

Code changes



Side effects



Related changes

 - Need to cherry-pick to the release branch
